xAPI.md

```diff
@@ -216,17 +216,22 @@ JSON allows for objects to have properties that contain empty objects. This is n
 * [Actor](#def-actor)
 * [Authentication](#def-authentication)
 * [Authorization](#def-authorization)
-* [Base Endpoint](#def-baseendpoint)
+* [Base Resource Path](#def-baseresourcepath)
 * [Community of Practice](#def-community-of-practice)
-* [Experience API (xAPI)](#def-experience-api)
+* [Experience Application Programming Interface (xAPI)](#def-experience-api)
```

Comment:
I'm *****extremely***** against this change. The thing this specification needs least is another name. 

Can we stick to "Experience API (or xAPI for short) aka 'Tin Can' or 'Tin Can API'" as the thing we have to say every time we talk to people about the spec and we're not sure what they call it?